lib/Controller/ApiController.php

```diff
@@ -465,12 +470,36 @@ public function getPolls() {
 		$eventsList = array();
 
 		foreach ($events as $eventElement) {
-			$eventsList[$eventElement->id] = $this->getEvent($eventElement->id);
+			$eventsList[] = $this->getPoll($eventElement->id);
 		}
 
 		return new DataResponse($eventsList, Http::STATUS_OK);
 	}
 
+	/**
+	 * @NoAdminRequired
+	 * @NoCSRFRequired
```

Comment:
There should be a CSRF check on this route.

Reply:
@juliushaertl Umpf. Have the CSFR thing generally on my to do, because I have to understand how to deal with this. Any how to available?

Reply:
Didn't thik, that it was so easy.... 😆